exonum-java-binding/core/src/test/java/com/exonum/binding/core/storage/indices/BaseProofMapIndexProxyIntegrationTestable.java

```diff
@@ -305,21 +225,19 @@ void getProof_FourEntryMap_FirstAndLastByte_Contains() {
   }
 
   @Test
-  void getProof_MultiEntryMapContains() {
+  void getMultiProof_MultiEntryMapContains() {
     runTestWithView(database::createFork, (map) -> {
-      List<MapEntry<HashCode, String>> entries = createSortedMapEntries();
+      List<MapEntry<HashCode, String>> entries = createMapEntries();
       putAll(map, entries);
 
-      for (MapEntry<HashCode, String> e : entries) {
-        assertThat(map, provesThatPresent(e.getKey(), e.getValue()));
-      }
+      assertThat(map, provesThatPresent(entries));
     });
   }
 
   @Test
   void getProof_MultiEntryMapDoesNotContain() {
```

Comment:
This test creates 32-byte keys, but it still seems to be appropriate for both proof map cases.